source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,36 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
+
+The entry number is unique and defines an entry’s position within the ordered list of changes. The entry timestamp is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The entry number is.
+
+Each entry is connected to an item by a given item hash. Entries reference items in the `item-hash` field using the hash derived from the corresponding item's content.
+
+Each key is a unique UTF-8 string which identifies something in a register. The `key` field holding this data always has the same name as the register. For example, every entry in the `local-authority-eng` register has a field called `local-authority-eng`.
```

Comment:
This is not quite accurate. It is the **item** rather than the **entry** that has a field called `local-authority-eng`. There is a `key` property in the **entry**. The data contained in this property happens to be the same as the data contained in the field of the corresponding **item** that has the same name as the register.

Might be worth splitting it out into the corresponding entry and item sections?

Entry: 
> Each key is a unique UTF-8 string which identifies something in a register. 

Item:
> The item always contains a field which has the same name as the register. For example, every item in the `local-authority-eng` register has a field called `local-authority-eng`. The data contained in this field will be the same as the data in the `key` property of the entry that introduced the item to the register.

This second statement is not true for indexes, but I assume we're ignoring their existence for now.

Reply:
Sounds good to me. I'll move the content around as you suggested in the next commit.